docs/0.4.0-alpha/getting_started/usage.md

```diff
@@ -142,6 +142,7 @@ source <(amber comp)
 
 | Icon | Name | Location |
 |---|:----:|:-----:|
-| ![image](/images/logo-vsc.png)  | **VS Code** | [VSC Marketplace](https://marketplace.visualstudio.com/items?itemName=Ph0enixKM.amber-language) or [Our extension repository](https://github.com/amber-lang/vsc-amber-extension) |
-| ![image](/images/logo-zed.png) | **Zed** | Zed extensions or [Our extension repository](https://github.com/amber-lang/zed-amber-extension) |
-| ![image](/images/logo-vim.png)  | **Vim** | [Our extension repository](https://github.com/amber-lang/amber-vim) |
+| LOGO:vsc | **VS Code** | [VSC Marketplace](https://marketplace.visualstudio.com/items?itemName=Ph0enixKM.amber-language) or [Our extension repository](https://github.com/amber-lang/vsc-amber-extension) |
+| LOGO:zed | **Zed** | Zed extensions or [Our extension repository](https://github.com/amber-lang/zed-amber-extension) |
+| LOGO:vim | **Vim** | [Our extension repository](https://github.com/amber-lang/amber-vim) |
+| LOGO:hx | **Helix Editor** | [Native](https://docs.helix-editor.com/lang-support.html) |
```

Comment:
> Native

I am not sure what word should be here. Feel free to suggest one.

Reply:
Native is perfect to me